features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/reactionsummary/ReactionSummaryView.kt

```diff
@@ -192,13 +199,24 @@ private fun AggregatedReactionButton(
             verticalAlignment = Alignment.CenterVertically,
             modifier = Modifier,
         ) {
-            Text(
-                text = reaction.displayKey,
-                style = ElementTheme.typography.fontBodyMdRegular.copy(
-                    fontSize = 20.sp,
-                    lineHeight = 25.sp
-                ),
-            )
+            if (reaction.key.startsWith("mxc://")) {
+                BlurHashAsyncImage(
+                    modifier = modifier
+                        .heightIn(min = reactionEmojiLineHeight.toDp(), max = reactionEmojiLineHeight.toDp())
```

Comment:
The line height used here should be `25.sp` instead (see the `Text` below). Maybe those should be extracted to a shared private value?

Reply:
Sorry, I actually meant that it's `25.sp` here while it's `20.sp` in the other file, so each one should have its own value.